### PR TITLE
yadm*: add pages

### PR DESCRIPTION
**Pull request by louiss0**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** 3.2.2

### CODE REVIEW

**Comment by louiss0** [comment]
The only error that should exist. If found is the one for. `TLDR010 Only Unix-style line endings allowed`

I don't know know If I should configure vs code  to account for this error.

If you want to do more research go to. <https://yadm.io/docs/overview>

**Comment by louiss0** [comment]
> Please read the [contributing](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md) and [style-guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md). Right now it does not seem to be okay according to our guides.

Any specific mistakes? I know that I added extra examples, but I still went below the example limit.

File: pages/common/yadm-git-crypt.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# yadm git-crypt
+
+> If git-crypt is installed, this command allows you to pass options directly to it.
+> With the environment configured to use the yadm repository.
+> Git Crypt enables transparent encryption and decryption of files in a git repository.
+> You can read <https://github.com/AGWA/yadm> git-crypt for details.
```
[the remark this answers is not in the document]
Comment (louiss0) [reply pages/common/yadm-git-crypt.md:6]: Nope that page doesn't exist

**Comment by louiss0** [comment]
Well this has been tough. I'm going to have to really read that style guide. I thought this would be simple.
Tldr lint didn't catch the errors that were talked about here.

**Comment by louiss0** [comment]
> There are still enough comments open. The linter can only check a set of the style guide. Please read the style guide carefully and apply it to the PR. Everyones first big contribution was hard.

Is there anything left for me to do? I think I'm done. I fixed all the links.

File: pages/common/yadm-list.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# yadm-list
+
+> This command allows you to print a list of files managed by yadm.
+> More information: <https://github.com/TheLocehiliosan/yadm/blob/master/yadm.md#commands>.
+
+- Execute:
+
+`yadm list`
+
+- The previous command only works with the root to list all:
```
[the remark this answers is not in the document]
Comment (louiss0) [reply pages/common/yadm-list.md:10]: No that the -a flag forces all files to be seen regardless of condition.

**Comment by louiss0** [comment]
> @louiss0 thank you for fixing some comments, but as long as there are open comments, please do not re-request my review.

I'm sorry but. You weren't responding to me in our conversations. I decided to use the ask button.

File: pages.en
[the remark this answers is not in the document]
Comment (louiss0) [reply pages.en:1]: It's causing me problems in my local repo. So i decided to get rid of it.
Sorry

**Comment by louiss0** [comment]
> > > @louiss0 thank you for fixing some comments, but as long as there are open comments, please do not re-request my review.
> > 
> > 
> > I'm sorry but. You weren't responding to me in our conversations. I decided to use the ask button.
> 
> Which conversation? Your last question was about if there was any work to do. Since there are open comments, there still is enough work to do.

I did change the list page. What can I do about the pages.en file? I'm not sure it exist on my local machine. I think I'll edit the rest of the files online from now until this is over.

**Comment by louiss0** [comment]
One thing I'm confused about is that you made me use the `yadm` symbol in many of the files. 
I wanted to use `yadm` to refer to the main command and **Yadm** to refer to yadm the CLI.
I guess people would understand the context, If I'm wrong then Why?

**Comment by louiss0** [comment]
> The [build](https://github.com/tldr-pages/tldr/actions/runs/11058131388) for this PR failed with the following error(s):
> 
> ```
> ./pages.en:
> Error: Lexical error on line 1. Unrecognized text.
> pages
> ^
> ./pages.en:0: TLDR107 File name should end with .md extension
> ```
> 
> Please fix the error(s) and push again.

@sebastiaanspeck Now I have this error. What do I do? I'm thinking about changing it into an md file.

**Comment by louiss0** [comment]
> > > > The [build](https://github.com/tldr-pages/tldr/actions/runs/11058131388) for this PR failed with the following error(s):
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > ./pages.en:
> > > 
> > > 
> > > > Error: Lexical error on line 1. Unrecognized text.
> > > 
> > > 
> > > > pages
> > > 
> > > 
> > > > ^
> > > 
> > > 
> > > > ./pages.en:0: TLDR107 File name should end with .md extension
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > Please fix the error(s) and push again.
> > > 
> > > 
> > > @sebastiaanspeck Now I have this error. What do I do? I'm thinking about changing it into an md file.
> > 
> > 
> > No, it is a symbolic link, which OS do you use? Let me help you to fix it.
> 
> Firstly remove your current pages.en and then use `ln -s pages pages.en` OR `mklink pages pages.en` (depending on your OS (`mklink` is for Windows)

I got around to it

**Comment by louiss0** [comment]
@sebastiaanspeck  I fixed the `pages.en` file.

**Comment by louiss0** [comment]
> Still open comments. Until they are resolved, my review stays the same.

Where?

**Comment by louiss0** [comment]
> > > Still open comments. Until they are resolved, my review stays the same.
> > 
> > Where?
> 
> If you go to the Files-tab, you will probably see them. There are only some files that don't have a comment, all other files have at least one comment.

What about the commits that I made on this page? Do they not count?

**Review by louiss0 — COMMENTED** [review]
This was a long journey. I shouldn't have rushed this project at all.

**Comment by louiss0** [comment]
> > [CLA assistant check](https://cla-assistant.io/tldr-pages/tldr?pullRequest=13799) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/tldr-pages/tldr?pullRequest=13799) before we can accept your contribution.**2** out of **3** committers have signed the CLA.✅ louiss0✅ sebastiaanspeck❌ louisshel
> > **louisshel** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> > You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/tldr-pages/tldr?pullRequest=13799) it.
> 
> Hi, can you please fix the commit that was committed with the wrong account?

louisshel is me sorry. How can I remove that one?

**Comment by louiss0** [comment]
> The [build](https://github.com/tldr-pages/tldr/actions/runs/11089357120) for this PR failed with the following error(s):
> 
> ```
> ./pages.en:
> Error: Lexical error on line 1. Unrecognized text.
> pages
> ^
> ./pages.en:0: TLDR107 File name should end with .md extension
> ```
> 
> Please fix the error(s) and push again.

@sebastiaanspeck  The pages.en file is annoying me again! I tried to use `ln pages.en pages/` that didn't work what now?

**Comment by louiss0** [comment]
I don't know what's going on with the CLA stuff, but I'm sure that I signed it. It even says it on the page.

image

**Comment by louiss0** [comment]
> See the open comments.

I'm new to this. I thought I took care of all the comments.

**Comment by louiss0** [comment]
@sebastiaanspeck I'm sorry I tried getting rid of louisshel but it won't let me. I tried to use git rebase -i then i tried to amend the commit in the rebase but that didn't work. i tried to amend while reverting that didn't work. I just don't know what's going on.

**Comment by louiss0** [comment]
> The [build](https://github.com/tldr-pages/tldr/actions/runs/11117095920) for this PR failed with the following error(s):
> 
> ```
> ./pages.en:
> Error: Lexical error on line 1. Unrecognized text.
> pages
> ^
> ./pages.en:0: TLDR107 File name should end with .md extension
> ```
> 
> Please fix the error(s) and push again.

@sebastiaanspeck This page is a problem again.

**Comment by louiss0** [comment]
@sebastiaanspeck I'm going do do a different PR. I'm sorry but I think this is the only way.
